### DISCUSSION TITLE
Converting expression to NAND etc

### DISCUSSION THREAD

**Discussion by dawinaj** [josdejong/mathjs#3068] [Q&A]
Hello,
is it possible to convert some expression to eg. NAND only? Using map() or anything else... Or maybe you could add it as a new feature if doesn't exist yet?

**Comment by josdejong** [comment]
Thanks for your suggestion @herhor67. I guess it would be easy to implement this as a simple logical operation. How would that work with `map`? The current logical operators (`and`, `or`, ...) work element-wise on matrices.

**Comment by dawinaj** [comment]
Don't know if I understand right. I didn't dig into what map can do, neither am I able to create function for this.
I started 2nd semester of studies, and manual converting is pain in the ass, so I thought maybe mathjs can do it much faster and without mistakes.

**Comment by josdejong** [comment]
@herhor67 can you explain your use case? What you have to calculate and how you think mathjs could help making your life easier?

**Comment by gwhitney** [comment]
Converting to a question in the discussions as it's not quite clear what's being asked for/proposed here.
